### ISSUE TITLE
Supersaw and V/Oct

### ISSUE THREAD

**Issue by the1andonlydrno** [Miserlou/RJModules#52]
Hi, I am about to compare all Supersaw oscillators for the next issue of "What does this knob do?" and by now I am struggling a few hours with your Supersaw. I tried a lot of things to make it play a sequencer line correctly. I even tried rescaling and rectifying the signal I put in the frequency cv input. The best result has still wrong intervals. Now I am out of ideas and a bit frustrated and pissed. Do I miss something or is this module not supposed to work with v/oct? Thanks for your efforts and modules

**Comment by Miserlou** [comment]
Yeah, Supersaw isn't v/oct. Ticket already opened here: https://github.com/Miserlou/RJModules/issues/32

Supersaw was the first module I ever made and it basically sucks. :(((( I might just deprecated it and make a new one, or try to fix it up, I'm not sure. PR welcome. I like the tone of it but without v/oct it's only really useful for drone.

**Comment by the1andonlydrno** [comment]
Thanks for your reply. In the meantime I found a way to make it work with Micro Map from stoermelder which moves the knob itself. Maybe this helps you in finding out how to fix it.

**Comment by squinkylabs** [comment]
There was some discussion of this issue on Facebook today. do you think there is something you could do to keep people from wasting their time trying to get it to work? A second sentence in the instruction manual? the word "deprecated" on the panel?
